lib/fennec/evaluator/allocate/request.ex

```diff
@@ -85,12 +85,14 @@ defmodule Fennec.Evaluator.Allocate.Request do
   end
 
   defp verify_unknown_attributes(params, state) do
-    case Params.get_attrs(params) do
-      [] ->
-        {:continue, params, state}
+    with u  <- Params.get_attr(params, Attribute.Username),
```

Comment:
You could use simple `case` expression here, since you're not matching on User nor Realm